src/allmydata/test/test_util.py

```diff
@@ -1,5 +1,11 @@
 from __future__ import print_function
```

Comment:
Since it looks like this module is now completely ported, please add a module docstring that says "Ported to Python 3."